manifests/charts/cloudcore/Chart.yaml

```diff
@@ -1,7 +1,22 @@
 apiVersion: v1
 name: cloudcore
+description: The KubeEdge cloudcore component.
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
 version: 0.1.0
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
 appVersion: 1.9.1
```

Comment:
do we need update this version since we have released version 1.11?

Reply:
Seems no need to update it every time. But once the chart have brought big changes, we had better upgrade it.